README.md

```
@@ -1,4 +1,4 @@
-# Synix Bot
+#  Synix Bot
```

markdownlint-cli2 (0.18.1)

Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)